riddlc/src/test/scala/com/reactific/riddl/RunRiddlcOnArbitraryTest.scala

```diff
@@ -36,5 +36,10 @@ class RunRiddlcOnArbitraryTest extends RunCommandSpecBase {
         succeed
       }
     }
+
+    s"run --show-times from example hugo" in {
+      val args = Seq("--show-times", "from", "riddlc/src/test/scala/com/reactific/riddl/example.conf", "hugo")
```

Comment:
Placing that example.conf in this location is incorrect for the design of this module. Please place it in the "input" folder in src/test/

Reply:
What is example.riddl trying to prove? Please change the "run --show-times from example hugo" to describe what's being tested here.